packages/design-tokens/src/js/device/index.ts

```diff
@@ -0,0 +1,166 @@
+// Breakpoint
+export const breakpointMobile = 0;
+export const breakpointTablet = '768px';
+export const breakpointDesktop = '1280px';
+
+// Form field
+export const formFieldSmallRadiusMobile = 'var(--radius-300)';
```

Comment:
please use directly the values, because Supernova will probably loose the connection. So find the actual value of radius-300 and use it instead.

 ```suggestion
export const formFieldSmallRadiusMobile = '8px';
...
```